packages/aws-cdk-lib/aws-dynamodb/lib/global-table.ts

```diff
@@ -0,0 +1,993 @@
+   *
+   * @default false
+   */
+  readonly deletionProtection?: boolean;
+
+  /**
+   * Whether point-in-time recovery is enabled on all replica tables in the global table.
+   *
+   * Note: This is configurable on a per-replica basis.
+   *
+   * @default false
+   */
+  readonly pointInTimeRecovery?: boolean;
+
+  /**
+   * The table class for all replica tables in the global table.
+   *
+   * Note: This is configurable on a per-replica basis.
+   *
+   * @default TableClass.STANDARD
+   */
+  readonly tableClass?: TableClass;
+}
+
+/**
+ * Properties used to configure a replica table
+ */
+export interface ReplicaTableProps extends TableOptionsV2 {
+  /**
+   * The region that the replica table will be created in.
+   */
+  readonly region: string;
+
+  /**
+   * The read capacity for the replica table.
+   *
+   * @default - replica table read capacity is inherited from read capacity defined at
+   * global table
+   */
+  readonly readCapacity?: Capacity;
+
+  /**
+   * Kinesis Data Stream to capture item-level changes for the replica table.
+   *
+   * @default - no Kinesis Data Stream
+   */
+  readonly kinesisStream?: IStream;
+
+  /**
+   * Options used to configure global secondary indexes on a per-replica basis.
+   *
+   * @default - global secondary index options configured on the global table level will
+   * be applied to the replica table
+   */
+  readonly globalSecondaryIndexOptions?: { [indexName: string]: ReplicaGlobalSecondaryIndexOptions };
+}
+
+/**
+ * Properties used to configure a global table
+ */
+export interface GlobalTableProps extends TableOptionsV2, SchemaOptions {
+  /**
+   * The name of all replica tables in the global table.
+   *
+   * @default - generated by CloudFormation
+   */
+  readonly tableName?: string;
+
+  /**
+   * The name of the TTL attribute for all replica tables in the global table.
+   *
+   * @default - TTL is disabled
+   */
+  readonly timeToLiveAttribute?: string;
+
+  /**
+   * The removal policy applied to the global table.
+   *
+   * @default RemovalPolicy.RETAIN
+   */
+  readonly removalPolicy?: RemovalPolicy;
+
+  /**
+   * The billing mode and the associated read and write capacity settings for all replica
+   * tables in the global table.
+   *
+   * Note: Read capacity is configurable on a per-replica basis.
+   *
+   * @default Billing.onDemand()
+   */
+  readonly billing?: Billing;
+
+  /**
+   * Properties used to configure the replica tables in the global table.
+   *
+   * Note: You can add or remove replicas after table creation, but you can only add or
+   * remove a single replica in each update.
+   *
+   * @default - a single replica table will be created in the region the global table is
+   * deployed to
+   */
+  readonly replicas?: ReplicaTableProps[];
+
+  /**
+   * Global secondary indexes to define on all replica tables in the global table.
+   *
+   * Note: Global secondary indexes can be configured on a per-replica basis. Tables only
```

Comment:
What does "configured" mean? Entire GSI definition, or just some aspects of it?

Reply:
Good point. Probably better wording would be something like "Read capacity and contributor insights can be configured on a per-replica basis"